### PR TITLE
fix: deno import from skypack

### PR DESCRIPTION
**Pull request by baoshan**
This PR should fix [this issue](https://github.com/octokit/octokit.js/issues/2185).

I’m not sure if `import type` alone is sufficient. It’s likely this PR will not be merged because there’s no maintainer currently. Anyway here it is.

### CODE REVIEW

**Comment by wolfy1339** [comment]
> It’s likely this PR will not be merged because there’s no maintainer currently.

There are some community maintainers that help around, though no full time active maintainer.

Thanks for the PR!

Can you instead put `aws-lambda` as an `optionalDependency`? I don't think it makes sense to always install it whenever a user installs this package

**Comment by baoshan** [comment]
Thanks @wolfy1339 for the suggestion!

It’s optional now. I’m interested if `import type` alone (without `optionalDependency`) is enough to silent the error. Will make some experiments when I have time. Actually `skypack` and `esm.sh` behave differently, it may be a long way to go before a satisfying Deno experience.

Since there are undergoing changes to Octokit.js, I guess the current repos will phase out. Maybe it’s best to be patient for now.

**Comment by wolfy1339** [comment]
Let's leave it as-is for now, merge this PR and revisit this if it becomes a problem again

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**
